README.md

```diff
@@ -1,56 +1,44 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Book Store
 
-# Project Name
-
-> Description the project.
+## Screenshot ![see](./screenshot.png?raw=true 'Title')
 
+> The app retrieves available tickers from the API lists them in an home page. When a particular ticker is clicker it routes to the information of that ticker.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- React
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
 ### Setup
 
+1. git clone https://github.com/mnekx/crypto-tickers.git
+2. cd crypto-tickers
+3. npm install
+4. npm start to serve into local 3000 port
+
 ### Install
 
-### Usage
+1. git clone https://github.com/mnekx/crypto-tickers.git
+2. cd crypto-tickers
+3. npm install
+4. npm start to serve into local 3000 port
 
 ### Run tests
 
-### Deployment
+npm run test
 
+### Deployment
 
+https://mnekx.github.io/crypto-tickers/
```

Comment:
- [ ] When I click on the **Live Demo Link**, this is what is shown 

![image](https://user-images.githubusercontent.com/77106962/163585052-166f8e18-37dd-4b84-b407-2b54fb24909d.png)


Kindly deploy with [**_Netlify_**](https://www.freecodecamp.org/news/how-to-deploy-react-router-based-app-to-netlify/#:~:text=The%20fastest%20and%20easy%20way,line%20from%20your%20project%20folder.) or deploy with [**_Heroku_**](https://www.geeksforgeeks.org/how-to-deploy-react-app-to-heroku/) if _**GitHub Pages**_ is hard to deploy with. 👍